File: 2219-maximum-number-of-words-found-in-sentences/solution.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        vector<int>nums; int ans;

        for(int i=0;i<sentences.size();++i){
            string word = sentences[i]; int x = 0;
            for(int j=0;j<word.length();++j){
                if(word[j] == ' ')x++;
            }
            nums.push_back(x);
        }
        sort(nums.begin(),nums.end());
        ans = nums[nums.size()-1]+1;
        return ans;
    }
};

File: 2219-maximum-number-of-words-found-in-sentences/solution.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        count_list = []
        
        for word in sentences:
          count = 0
          new_str = word.split()
          
          for i in new_str:
            count +=1
          count_list.append(count)
          
        return max(count_list)
